src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    operate(callback, value) {
+      if (!isNaN(value)) {
```

Comment:
There is no need to check whether the value is not a number. It's better to verify if the provided argument is indeed a number for better type safety.